.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "feast-devcontainer",
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      "version": "latest"
+    },
+    "ghcr.io/devcontainers/features/python:1": {
+      "version": "3.9"
+    },
+    "ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {
+      "version": "latest"
+    }
```

Comment:
What does this layer do?

Reply:
It installs brew, so I can easily install mysql :)